docs/overview.md

```diff
@@ -0,0 +1 @@
+# Overview here
```

Comment:
Fill in with actual documentation, I think we just include it directly in this PR.